### PR TITLE
Updates to FLORDM cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Updated FLORDM manuf sn's per metadata review.  (sensor bulk load also updated on PR#436)

### CODE REVIEW

**Review by snwhite — APPROVED** [review]
All files reviewed and look good.

File: calibration/FLORDM/CGINS-FLORDM-03061__20130430.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3061,CC_angular_resolution,1.096,
-FLBBSLC-3061,CC_depolarization_ratio,0.039,
-FLBBSLC-3061,CC_measurement_wavelength,700.0,
-FLBBSLC-3061,CC_scattering_angle,140.0,
+3061,CC_angular_resolution,1.096,
+3061,CC_depolarization_ratio,0.039,
+3061,CC_measurement_wavelength,700.0,
+3061,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03061__20130430.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03063__20130429.csv [hunk 2]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3063,CC_angular_resolution,1.096,
-FLBBSLC-3063,CC_depolarization_ratio,0.039,
-FLBBSLC-3063,CC_measurement_wavelength,700.0,
-FLBBSLC-3063,CC_scattering_angle,140.0,
+3063,CC_angular_resolution,1.096,
+3063,CC_depolarization_ratio,0.039,
+3063,CC_measurement_wavelength,700.0,
+3063,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03063__20130429.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03063__20160210.csv [hunk 3]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3063,CC_angular_resolution,1.096,
-FLBBSLC-3063,CC_depolarization_ratio,0.039,
-FLBBSLC-3063,CC_measurement_wavelength,700.0,
-FLBBSLC-3063,CC_scattering_angle,140.0,
+3063,CC_angular_resolution,1.096,
+3063,CC_depolarization_ratio,0.039,
+3063,CC_measurement_wavelength,700.0,
+3063,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03063__20160210.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03063__20170707.csv [hunk 4]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3063,CC_angular_resolution,1.096,
-FLBBSLC-3063,CC_depolarization_ratio,0.039,
-FLBBSLC-3063,CC_measurement_wavelength,700.0,
-FLBBSLC-3063,CC_scattering_angle,140.0,
+3063,CC_angular_resolution,1.096,
+3063,CC_depolarization_ratio,0.039,
+3063,CC_measurement_wavelength,700.0,
+3063,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03063__20170707.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03064__20130426.csv [hunk 5]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3064,CC_angular_resolution,1.096,
-FLBBSLC-3064,CC_depolarization_ratio,0.039,
-FLBBSLC-3064,CC_measurement_wavelength,700.0,
-FLBBSLC-3064,CC_scattering_angle,140.0,
+3064,CC_angular_resolution,1.096,
+3064,CC_depolarization_ratio,0.039,
+3064,CC_measurement_wavelength,700.0,
+3064,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03064__20130426.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03123__20130503.csv [hunk 6]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3123,CC_angular_resolution,1.096,
-FLBBSLC-3123,CC_depolarization_ratio,0.039,
-FLBBSLC-3123,CC_measurement_wavelength,700.0,
-FLBBSLC-3123,CC_scattering_angle,140.0,
+3123,CC_angular_resolution,1.096,
+3123,CC_depolarization_ratio,0.039,
+3123,CC_measurement_wavelength,700.0,
+3123,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03123__20130503.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03123__20160210.csv [hunk 7]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3123,CC_angular_resolution,1.096,
-FLBBSLC-3123,CC_depolarization_ratio,0.039,
-FLBBSLC-3123,CC_measurement_wavelength,700.0,
-FLBBSLC-3123,CC_scattering_angle,140.0,
+3123,CC_angular_resolution,1.096,
+3123,CC_depolarization_ratio,0.039,
+3123,CC_measurement_wavelength,700.0,
+3123,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03123__20160210.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03124__20130503.csv [hunk 8]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3124,CC_angular_resolution,1.096,
-FLBBSLC-3124,CC_depolarization_ratio,0.039,
-FLBBSLC-3124,CC_measurement_wavelength,700.0,
-FLBBSLC-3124,CC_scattering_angle,140.0,
+3124,CC_angular_resolution,1.096,
+3124,CC_depolarization_ratio,0.039,
+3124,CC_measurement_wavelength,700.0,
+3124,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03124__20130503.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03124__20141023.csv [hunk 9]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3124,CC_angular_resolution,1.096,
-FLBBSLC-3124,CC_depolarization_ratio,0.039,
-FLBBSLC-3124,CC_measurement_wavelength,700.0,
-FLBBSLC-3124,CC_scattering_angle,140.0,
+3124,CC_angular_resolution,1.096,
+3124,CC_depolarization_ratio,0.039,
+3124,CC_measurement_wavelength,700.0,
+3124,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03124__20141023.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03124__20160627.csv [hunk 10]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3124,CC_angular_resolution,1.096,
-FLBBSLC-3124,CC_depolarization_ratio,0.039,
-FLBBSLC-3124,CC_measurement_wavelength,700.0,
-FLBBSLC-3124,CC_scattering_angle,140.0,
+3124,CC_angular_resolution,1.096,
+3124,CC_depolarization_ratio,0.039,
+3124,CC_measurement_wavelength,700.0,
+3124,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03124__20160627.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03124__20190827.csv [hunk 11]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3124,CC_angular_resolution,1.096,
-FLBBSLC-3124,CC_depolarization_ratio,0.039,
-FLBBSLC-3124,CC_measurement_wavelength,700.0,
-FLBBSLC-3124,CC_scattering_angle,140.0,
+3124,CC_angular_resolution,1.096,
+3124,CC_depolarization_ratio,0.039,
+3124,CC_measurement_wavelength,700.0,
+3124,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03124__20190827.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03527__20140322.csv [hunk 12]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3527,CC_angular_resolution,1.096,
-FLBBSLC-3527,CC_depolarization_ratio,0.039,
-FLBBSLC-3527,CC_measurement_wavelength,700.0,
-FLBBSLC-3527,CC_scattering_angle,140.0,
+3527,CC_angular_resolution,1.096,
+3527,CC_depolarization_ratio,0.039,
+3527,CC_measurement_wavelength,700.0,
+3527,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03527__20140322.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03527__20160210.csv [hunk 13]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3527,CC_angular_resolution,1.096,
-FLBBSLC-3527,CC_depolarization_ratio,0.039,
-FLBBSLC-3527,CC_measurement_wavelength,700.0,
-FLBBSLC-3527,CC_scattering_angle,140.0,
+3527,CC_angular_resolution,1.096,
+3527,CC_depolarization_ratio,0.039,
+3527,CC_measurement_wavelength,700.0,
+3527,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03527__20160210.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03536__20160802.csv [hunk 14]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3536,CC_angular_resolution,1.096,
-FLBBSLC-3536,CC_depolarization_ratio,0.039,
-FLBBSLC-3536,CC_measurement_wavelength,700,
-FLBBSLC-3536,CC_scattering_angle,140,
+3536,CC_angular_resolution,1.096,
+3536,CC_depolarization_ratio,0.039,
+3536,CC_measurement_wavelength,700,
+3536,CC_scattering_angle,140,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03536__20160802.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03540__20140409.csv [hunk 15]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBCDSLC-3540,CC_angular_resolution,1.096,
-FLBBCDSLC-3540,CC_depolarization_ratio,0.039,
-FLBBCDSLC-3540,CC_measurement_wavelength,700.0,
-FLBBCDSLC-3540,CC_scattering_angle,140.0,
+3540,CC_angular_resolution,1.096,
+3540,CC_depolarization_ratio,0.039,
+3540,CC_measurement_wavelength,700.0,
+3540,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03540__20140409.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03549__20140430.csv [hunk 16]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBCDSLC-3549,CC_angular_resolution,1.096,
-FLBBCDSLC-3549,CC_depolarization_ratio,0.039,
-FLBBCDSLC-3549,CC_measurement_wavelength,700,
-FLBBCDSLC-3549,CC_scattering_angle,140,
+3549,CC_angular_resolution,1.096,
+3549,CC_depolarization_ratio,0.039,
+3549,CC_measurement_wavelength,700,
+3549,CC_scattering_angle,140,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03549__20140430.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03549__20150817.csv [hunk 17]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBCDSLC-3549,CC_angular_resolution,1.096,
-FLBBCDSLC-3549,CC_depolarization_ratio,0.039,
-FLBBCDSLC-3549,CC_measurement_wavelength,700.0,
-FLBBCDSLC-3549,CC_scattering_angle,140.0,
+3549,CC_angular_resolution,1.096,
+3549,CC_depolarization_ratio,0.039,
+3549,CC_measurement_wavelength,700.0,
+3549,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03549__20150817.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03550__20140430.csv [hunk 18]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3550,CC_angular_resolution,1.096,
-FLBBSLC-3550,CC_depolarization_ratio,0.039,
-FLBBSLC-3550,CC_measurement_wavelength,700,
-FLBBSLC-3550,CC_scattering_angle,140,
+3550,CC_angular_resolution,1.096,
+3550,CC_depolarization_ratio,0.039,
+3550,CC_measurement_wavelength,700,
+3550,CC_scattering_angle,140,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03550__20140430.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03550__20160210.csv [hunk 19]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3550,CC_angular_resolution,1.096,
-FLBBSLC-3550,CC_depolarization_ratio,0.039,
-FLBBSLC-3550,CC_measurement_wavelength,700.0,
-FLBBSLC-3550,CC_scattering_angle,140.0,
+3550,CC_angular_resolution,1.096,
+3550,CC_depolarization_ratio,0.039,
+3550,CC_measurement_wavelength,700.0,
+3550,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03550__20160210.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03552__20140430.csv [hunk 20]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3552,CC_angular_resolution,1.096,
-FLBBSLC-3552,CC_depolarization_ratio,0.039,
-FLBBSLC-3552,CC_measurement_wavelength,700.0,
-FLBBSLC-3552,CC_scattering_angle,140.0,
+3552,CC_angular_resolution,1.096,
+3552,CC_depolarization_ratio,0.039,
+3552,CC_measurement_wavelength,700.0,
+3552,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03552__20140430.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03553__20140430.csv [hunk 21]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3553,CC_angular_resolution,1.096,
-FLBBSLC-3553,CC_depolarization_ratio,0.039,
-FLBBSLC-3553,CC_measurement_wavelength,700.0,
-FLBBSLC-3553,CC_scattering_angle,140.0,
+3553,CC_angular_resolution,1.096,
+3553,CC_depolarization_ratio,0.039,
+3553,CC_measurement_wavelength,700.0,
+3553,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03553__20140430.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03554__20140430.csv [hunk 22]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3554,CC_angular_resolution,1.096,
-FLBBSLC-3554,CC_depolarization_ratio,0.039,
-FLBBSLC-3554,CC_measurement_wavelength,700.0,
-FLBBSLC-3554,CC_scattering_angle,140.0,
+3554,CC_angular_resolution,1.096,
+3554,CC_depolarization_ratio,0.039,
+3554,CC_measurement_wavelength,700.0,
+3554,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03554__20140430.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03578__20140721.csv [hunk 23]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBCDSLC-3578,CC_angular_resolution,1.096,
-FLBBCDSLC-3578,CC_depolarization_ratio,0.039,
-FLBBCDSLC-3578,CC_measurement_wavelength,700.0,
-FLBBCDSLC-3578,CC_scattering_angle,140.0,
+3578,CC_angular_resolution,1.096,
+3578,CC_depolarization_ratio,0.039,
+3578,CC_measurement_wavelength,700.0,
+3578,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03578__20140721.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03579__20140603.csv [hunk 24]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3579,CC_angular_resolution,1.096,
-FLBBSLC-3579,CC_depolarization_ratio,0.039,
-FLBBSLC-3579,CC_measurement_wavelength,700.0,
-FLBBSLC-3579,CC_scattering_angle,140.0,
+3579,CC_angular_resolution,1.096,
+3579,CC_depolarization_ratio,0.039,
+3579,CC_measurement_wavelength,700.0,
+3579,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03579__20140603.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03580__20140603.csv [hunk 25]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3580,CC_angular_resolution,1.096,
-FLBBSLC-3580,CC_depolarization_ratio,0.039,
-FLBBSLC-3580,CC_measurement_wavelength,700.0,
-FLBBSLC-3580,CC_scattering_angle,140.0,
+3580,CC_angular_resolution,1.096,
+3580,CC_depolarization_ratio,0.039,
+3580,CC_measurement_wavelength,700.0,
+3580,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03580__20140603.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03582__20140721.csv [hunk 26]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3582,CC_angular_resolution,1.096,
-FLBBSLC-3582,CC_depolarization_ratio,0.039,
-FLBBSLC-3582,CC_measurement_wavelength,700.0,
-FLBBSLC-3582,CC_scattering_angle,140.0,
+3582,CC_angular_resolution,1.096,
+3582,CC_depolarization_ratio,0.039,
+3582,CC_measurement_wavelength,700.0,
+3582,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03582__20140721.csv:5]: Looks good

File: calibration/FLORDM/CGINS-FLORDM-03582__20181213.csv [hunk 27]
```diff
@@ -1,5 +1,5 @@
 serial,name,value,notes
-FLBBSLC-3582,CC_angular_resolution,1.096,
-FLBBSLC-3582,CC_depolarization_ratio,0.039,
-FLBBSLC-3582,CC_measurement_wavelength,700.0,
-FLBBSLC-3582,CC_scattering_angle,140.0,
+3582,CC_angular_resolution,1.096,
+3582,CC_depolarization_ratio,0.039,
+3582,CC_measurement_wavelength,700.0,
+3582,CC_scattering_angle,140.0,
```
Comment (cdobs) [inline calibration/FLORDM/CGINS-FLORDM-03582__20181213.csv:5]: Looks good

**Review by cdobs — APPROVED** [review]

**Pull request merged**